src/librustc_apfloat/lib.rs

```diff
@@ -53,34 +53,30 @@
 #![cfg_attr(not(stage0), feature(const_min_value))]
 #![cfg_attr(not(stage0), feature(const_max_value))]
 
+#[allow(unused_extern_crates)]
+extern crate rustc_cratesio_shim;
```

Comment:
Can you add a comment here pointing to the `Cargo.toml` of rustc_cratesio_shim?

Reply:
Done.